routes/file.js

```diff
@@ -1,15 +1,46 @@
 const router = require("express").Router();
+const sharp = require("sharp");
+const { writeFileSync } = require("fs");
+
 const saveImages = require("../services/MulterService");
 
+/** Size to which the image is to be resized */
+const IMAGE_SIZE = {
+  height: 1080,
+  width: 720,
+};
+
+// Route to store images
 router.post("/", saveImages, (req, res) => {
   const files = req.files;
-  res.json({
-    files: files.map((file) => ({
-      mimeType: file.mimetype,
-      path: file.path,
-      createdOn: Date.now(),
-    })),
-  });
+
+  try {
+    files.forEach(async (file) => {
+      const { path } = file;
+
+      // Resizing image
+      const buffer = await sharp(path)
+        .resize(IMAGE_SIZE.height, IMAGE_SIZE.width)
+        .jpeg({ quality: 90 })
```

Comment:
reduce quality even more. We want less size of file.

Reply:
Specify a value for quality

Reply:
Make it 60-70 maybe